core/src/syscall/precompiles/keccak256/air.rs

```diff
@@ -64,34 +65,15 @@ where
         );
 
         // Constrain that the inputs stay the same throughout the 24 rows of each cycle
-        let mut not_last_in_cycle = local.keccak.step_flags[0].into();
```

Comment:
My `not_final_step` is computed differently, but the two should be guaranteed to give the same result based on the `eval_round_flags` constraints in `KeccakAir`.